### PR TITLE
feat(activity list): combine all activities into single feed

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4/src/redux/data/activity/reducers.ts [hunk 1]
```diff
@@ -41,54 +49,67 @@ export const activityReducer = (
...
-            ...state[action.payload.product].transactions,
-            items: [
-              Remote.Loading,
-              ...state[action.payload.product].transactions.items
-            ]
-          }
-        } as CustodialActivityType
+        [action.payload.product]: failure as CustodialActivityType
       }
+    // ❓
```
Comment (plondon) [inline packages/blockchain-wallet-v4/src/redux/data/activity/reducers.ts:66]: @schnogz I'd love to hear your thoughts on below 👇

Comment (plondon) [reply packages/blockchain-wallet-v4/src/redux/data/activity/reducers.ts:66]: https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/2698/commits/4b07a6aab81942e0077e3c37ba17f7bd730cbfbf#diff-079a24ed23a3c2be028c6e1813529bffR27-R30

My solution, I separated the data from the status of that data.

File: packages/blockchain-wallet-v4-frontend/src/scenes/Activity/selectors.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// import { ExtractSuccess } from 'core/types'
+// import { lift } from 'ramda'
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Activity/selectors.ts:2]: minor - maybe remove comments, we always can get them back from git history

File: packages/blockchain-wallet-v4/src/redux/data/activity/selectors.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    for (const type of NabuTxType) {
+      statuses.push(state.dataPath.activity[product][type].status)
+    }
+  }
+
+  return liftN(
+    NabuProducts.length * NabuTxType.length,
+    (...args) => args
+  )(...statuses)
+}
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4/src/redux/data/activity/selectors.ts:31]: I like this; to be honest it took me a bit to get how it works

Comment (plondon) [reply packages/blockchain-wallet-v4/src/redux/data/activity/selectors.ts:31]: What do you like about it?

Comment (milan-bc) [reply packages/blockchain-wallet-v4/src/redux/data/activity/selectors.ts:31]: isolation/separation of statuses

**Pull request closed without merging**